### PR TITLE
Troubleshoot docs refresh

### PR DESCRIPTION
**Pull request by usha-mandya**
Signed-off-by: Usha Mandya <usha.mandya@docker.com>

Update Docker Desktop Troubleshoot docs

### CODE REVIEW

File: docker-for-windows/troubleshoot.md [hunk 1]
```diff
@@ -160,15 +159,26 @@ script](https://github.com/moby/moby/issues/24388).
...
 
-2. Virtualization enabled in the BIOS
+1. Virtual Machine Platform
+2. [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10){:target="_blank" rel="noopener" class="_"}
+3. Virtualization enabled in the BIOS
+4. Hypervisor enabled at Windows startup
+
+![WSL 2 enabled](images/wsl2-enabled.png){:width="600px"}
+
+On **Windows 10**, you can also use Hyper-V with the following features enabled:
```
Comment (StefanScherer) [inline docker-for-windows/troubleshoot.md:173]: Maybe move the Hyper-V headline from line 182 to this line.

#### Hyper-V

On Windows 10 Pro or Enterprise,  you can also use Hyper-V with the following features enabled:

File: docker-for-windows/troubleshoot.md [hunk 2]
```diff
@@ -160,15 +159,26 @@ script](https://github.com/moby/moby/issues/24388).
...
 
+1. [Hyper-V](https://docs.microsoft.com/en-us/windows-server/virtualization/hyper-v/hyper-v-technology-overview){:target="_blank" rel="noopener" class="_"}
+   installed and working
+2. Virtualization enabled in the BIOS
 3. Hypervisor enabled at Windows startup
 
+![Hyper-V on Windows features](images/hyperv-enabled.png){:width="600px"}
+
 #### Hyper-V
```
Comment (StefanScherer) [inline docker-for-windows/troubleshoot.md:182]: move the headline between WSL 2 and Hyper-V

**Review by StefanScherer — COMMENTED** [review]
Looks good, I added only a few suggestions.

File: docker-for-windows/troubleshoot.md [hunk 3]
```diff
@@ -160,15 +159,26 @@ script](https://github.com/moby/moby/issues/24388).
 
 ### Virtualization
 
- Your machine must have the following features for Docker Desktop to function correctly:
+Your machine must have the following features for Docker Desktop to function correctly.
 
-1. [Hyper-V](https://docs.microsoft.com/en-us/windows-server/virtualization/hyper-v/hyper-v-technology-overview)
-   installed and working
+On **WSL 2** and **Windows Home**:
```
[the remark this answers is not in the document]
Comment (usha-mandya) [reply docker-for-windows/troubleshoot.md:164]: That makes sense. Thanks

File: docker-for-windows/troubleshoot.md (hunk 1 above)
Comment (usha-mandya) [reply docker-for-windows/troubleshoot.md]: Updated

**Review by ebriney — APPROVED** [review]

**Review by StefanScherer — APPROVED** [review]
LGTM

**Pull request merged**